src/en/wizden-staff/maintainer/adding-new-maintainers.md

```diff
@@ -0,0 +1,46 @@
+# Adding New Maintainers
+
+This document defines the process for adding new Maintainers to Space Station 14.
+
+## Requirements
+
+Maintainers represent Space Wizards as staff. Certain requirements must be met before a person can become a Maintainer:
+- A vote must be held according to the Voting Process outlined below.
+- Be a member of the Space Station 14 Discord server.
+- If the nominee has caused Administration issues in the past, these issues must be raised and discussed during the voting process. Administration issues include, but are not limited to, being banned from the game, banned from the Discord server, or having a history of being rude to other members of the community.
+- Have a good understanding of the Space Station 14 codebase and how it works. This must be raised and discussed during the voting process.
+- Have a solid understanding of the culture and values of the game and community
+
+### Voting Process
+
+To add a new Maintainer, a current Maintainer has to nominate the candidate. Once a person has been nominated, the following process should be followed:
+1. A vote will be created in the "Internal" category on the Space Station 14 Discourse, using the appropriate template. This vote will be open for one week, and all Maintainers will be notified of the vote.
+2. During this time, Maintainers should discuss the nomination, raise concerns, and ask any questions they may have. If there is a clear majority of Maintainers in favor or against the nomination, the vote may be closed early.
+    - If there is no clear majority, the vote must remain open for the full week.
+    - Concerns about a nominated person must be addressed before the vote may close. This will delay the vote until the concerns are resolved.
+    - For a vote to pass, it must have a supermajority (66%) of votes in favor. 
+        - Abstain votes are considered neutral. However, if counting all abstain votes as "No" would cause the vote to fall below the required 66% threshold, the vote fails.
```

Comment:
I dont understand how its an "abstain" if it still counts for the voting and can cause the vote to fail. Voting to abstain conventionally just means that you are formally not voting for either side, and i see no reason why it should be different here.